src/Security/MemberAuthenticator/LostPasswordHandler.php

```diff
@@ -24,108 +37,120 @@ class MemberLoginHandler extends FormRequestHandler
      * @config
      */
     private static $allowed_actions = [
-        'dologin',
-        'logout',
+        'lostpassword',
+        'LostPasswordForm',
+        'passwordsent',
     ];
 
+    private $link = null;
+
     /**
-     * Login form handler method
-     *
-     * This method is called when the user clicks on "Log in"
-     *
-     * @param array $data Submitted data
-     * @return HTTPResponse
+     * @param $link The URL to recreate this request handler
      */
-    public function dologin($data)
+    public function __construct($link)
     {
-        if ($this->performLogin($data)) {
-            return $this->logInUserAndRedirect($data);
-        }
+        $this->link = $link;
```

Comment:
> Changing it to link() is 100x less impact than changing it to getLink(). So we'll probably start with that.

I concur with @sminnee. Given that link($action) is a non-basic getter (it takes an argument), it doesn't fit the convention for get<property> methods.

However, for this pull request, please keep consistent case, unless you are willing to rename the method framework-wide. :)